packages/manager/apps/pci-volume-snapshot/src/api/data/snapshots.ts

```diff
@@ -0,0 +1,70 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+
+export interface TSnapshot {
+  id: string;
+  creationDate: string;
+  name: string;
+  description: string;
+  size: number;
+  volumeId: string;
+  region: string;
+  status: 'available' | 'creating' | 'deleting' | 'error' | 'error_deleting';
+  planCode: string | null;
+}
+
+export interface TVolume {
+  id: string;
+  attachedTo: string[];
+  creationDate: string;
+  name: string;
+  description: string;
+  size: number;
+  status:
+    | 'attaching'
+    | 'available'
+    | 'awaiting-transfer'
+    | 'backing-up'
+    | 'creating'
+    | 'deleting'
+    | 'detaching'
+    | 'downloading'
+    | 'error'
+    | 'error_backing-up'
+    | 'error_deleting'
+    | 'error_extending'
+    | 'error_restoring'
+    | 'extending'
+    | 'in-use'
+    | 'maintenance'
+    | 'reserved'
+    | 'restoring-backup'
+    | 'retyping'
+    | 'uploading';
+  region: string;
+  bootable: boolean;
+  planCode: string | null;
+  availabilityZone: string | null;
+  type:
+    | 'classic'
+    | 'classic-BETA'
+    | 'high-speed'
+    | 'high-speed-BETA'
+    | 'high-speed-gen2';
```

Comment:
Prefer having a fallback to string for type and status since they will change in the future